Libraries/StyleSheet/StyleSheet.js

```diff
@@ -13,7 +13,16 @@
 
 var StyleSheetRegistry = require('StyleSheetRegistry');
 var StyleSheetValidation = require('StyleSheetValidation');
-var flattenStyle = require('flattenStyle');
+
+/* TODO(brentvatne) docs are needed for this */
```

Comment:
Sure. That comment was already there on the `flatten` export, I was just trying to find a better place to put it that wouldn't show up in the docs.